Add more-info about unhealthy setups (#15729)
home-assistant/home-assistant.github.io: source/more-info/unhealthy/supervisor.markdown

---
title: "Supervisor was not able to update"

> description: "More information on why a failed Supervisor update mark a system as unhealthy."

Reviewer: ```suggestion
description: "More information on why a failed Supervisor update marks a system as unhealthy."
```

---

## The issue

> This can happen if there was a network issue during the startup of the Supervisor

Reviewer: ```suggestion
This can happen when there was a network issue during the startup of the Supervisor.
```

## The solution

> Manually update the Supervisor. This can be done from the "System" tab in the Supervisor panel, on the card for "Supervisor" there is a button to update the Supervisor.

Reviewer: ```suggestion
Manually update the Supervisor. This can be done from the "System" tab in the Supervisor panel. On the card for "Supervisor", there is a button to update the Supervisor.
```

> This can also be done with the CLI with `ha supervisor update`.

Reviewer: ````suggestion
This can also be done with the CLI, by running the following command:
```bash
ha supervisor update
```
````
